src/main/java/com/faforever/client/fx/PlatformService.java

```diff
@@ -130,4 +136,13 @@ private String getForegroundWindowTitle() {
   public boolean isWindowFocused(String windowTitle) {
     return windowTitle.equals(getForegroundWindowTitle());
   }
+
+  public void setUnixExecutableBit(Path exePath) throws IOException {
+	// client needs executable bit for running and the writeable bit set to alter the version
```

Comment:
We dont do any comments, only if there is really something to explain. Better call your functions and varibles in a proper understandable way.
If for example you wanna lay enphasis on the writable part call the function setUnixExecutableAndWritableBit

Reply:
The comment explains why the writable bit is necessary which is not obvious.